src/main/setup/steps/add-content/generate-document.ts

```diff
@@ -1,21 +1,46 @@
 import endent from "endent"
-import { ValidCNSInputs } from "../../../create-next-stack-types"
+import type {
+  ValidCNSInputs,
+  ValidCreateNextStackFlags,
+} from "../../../create-next-stack-types"
+
+const getImports = (flags: ValidCreateNextStackFlags) => {
+  if (flags.chakra) {
+    return endent/* tsx */ `import { ColorModeScript } from "@chakra-ui/react";`
+  } else {
+    return ""
+  }
+}
+
+const getHeadElements = (flags: ValidCreateNextStackFlags) => {
+  if (flags["material-ui"]) {
+    return endent/* tsx */ `<meta name="theme-color" content={theme.palette.primary.main} />
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+          />`
```

Comment:
Just a bit of formatting. `endent` allows us to write the backticks on their own lines, that is `endent` trims newlines at the start and end of a template string literal.
```suggestion
    return endent/* tsx */ `
      <meta name="theme-color" content={theme.palette.primary.main} />
      <link
        rel="stylesheet"
        href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
      />
    `
```